docs/releases/3.0.txt

```diff
@@ -213,6 +213,15 @@ backends.
 
 * ``DatabaseIntrospection.get_field_type()`` may no longer return tuples.
 
+* ``DatabaseOperations._prepare_tzname_delta()`` is used to customize the database spesific time zone delta function.
```

Comment:
I don't think this block quite makes sense as it is: `_prepare_tzname_delta()` is a private helper used only in the already private `_convert_field_to_tz()` — there's no guarantee at all that a backend will implement either. (The SQLite backend doesn't, for example...) 

Maybe just something like: 

```
Supported database backends now correctly handle ``timezone`` formats when
created using `timedelta` instances:

>>> from datetime import timezone, timedelta
>>> str(timezone(timedelta(hours=5)))
'UTC+05:00'

Third-party backends should ensure that they correctly handle this format when
preparing ``datetime`` fields in ``datetime_cast_date_sql()`` & co.
```

(@felixxm, @timgraham?) 

Docs should be wrapped to 79 chars.